redirects.yaml

```diff
@@ -1,5 +1,5 @@
-get-started: "/docs/installation"
-install: "/docs/installation"
+get-started: "/docs/snap/3.1/ui/maas-installation"
```

Comment:
This adds to the workload on release. Can we not maintain the latest version via the discourse redirect system?

Reply:
It does, but that's a one-time cost, vs a every-single-visit cost.

See below for the painful experience we put our users through currently...
```
$ time curl -L https://maas-io-678.demos.haus/docs/installation > /dev/null
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   277  100   277    0     0    122      0  0:00:02  0:00:02 --:--:--   122
100 44185  100 44185    0     0   9880      0  0:00:04  0:00:04 --:--:-- 36729

real    0m4.480s
user    0m0.018s
sys     0m0.000s
```

... compared to
```
$ time curl -L https://maas-io-678.demos.haus/install > /dev/null
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   277  100   277    0     0   5893      0 --:--:-- --:--:-- --:--:--  5893
100 44185  100 44185    0     0  19576      0  0:00:02  0:00:02 --:--:-- 42000

real    0m2.266s
user    0m0.011s
sys     0m0.007s
```

which is still approximately 2s slower than it *should* be, but better